wafer/templates/markitup/editor.html

```diff
@@ -0,0 +1,4 @@
+<div class="django-markitup-config" style="display: none"
+     data-element="#{{ textarea_id }}"
+     data-preview-url="{{ preview_url }}"
+     data-auto-preview="{{ AUTO_PREVIEW|yesno:"1,0" }}"></div>
```

Comment:
How does this change interact with Markitup's Admin widget?

Won't we need to also ensure the javascript snippet is included on the admin pages so this works there too?

Reply:
Good question. I wanted to test that too :)

Reply:
Resolved that by hooking into `wafer/compare/templates/admin/wafer.compare/change_form.html`. But that's a temporary hack if upstream will accept a similar change.

Reply:
Playing with this branch, it looks like we aren't doing the right thing for the markup fields used in the sponsors admin page, but otherwise this looks good.

Reply:
Overrode that admin template too.